python-package/lightgbm/plotting.py

```diff
@@ -513,7 +516,12 @@ def add(
                         missing_type_str=root['missing_type'],
                         default_left=root['default_left']
                     )
-            label += f"<B>{_float2str(root['threshold'], precision)}</B>"
+            if max_category_values is not None:
+                if root['decision_type'] == '==' and len(root['threshold'].split('||')) > max_category_values:
+                    tooltip = root['threshold']
+                    threshold = '...'
```

Comment:
I didn't really think this value through, maybe it would be a better user experience if it was something like "collapsed". What do you think?

Reply:
I don't think that I am following what do you mean by "collapsed", do you mean setting "collapsed" to ```threshold``` instead of the dots?

Reply:
Yes, I'm not sure if the dots are clear enough for users to try to hover over them. Maybe the text "collapsed" is clearer or something that would signal to the users that in order to see the actual thresholds they need to hover over the node. Could be enough with the docstring though. What do you think?

Reply:
I thought of something like turning "1001||1029||2384||2383||3484" into "1001||1029..." or "1001||1029...||3484"

Reply:
I think that's definitely better, maybe just have the dots between the ors? Like "1001||1029||...||3484"

Reply:
+1 to having `...` in the middle, I like that!

Reply:
great, I committed that change